server/athenian/api/controllers/team_controller.py

```diff
@@ -14,7 +22,25 @@ async def create_team(request: web.Request,
 
     """
     body = TeamCreateRequest.from_dict(body)
-    return web.Response(status=200)
+    user = request.uid
+    account = body.account
+    name = body.name
+    async with request.sdb.connection() as sdb_conn:
+        try:
+            await get_user_account_status(user, account, sdb_conn, request.cache)
+        except ResponseError as e:
+            return e.response
+        try:
+            members = _check_members(body.members)
+        except ResponseError as e:
+            return e.response
+        t = Team(owner=account, name=name, members=members).create_defaults()
+        try:
+            tid = await sdb_conn.execute(insert(Team).values(t.explode()))
+        except (UniqueViolationError, IntegrityError, OperationalError) as err:
+            return ResponseError(DatabaseConflict(
+                detail=f"this team already exists: {str(err)}")).response
```

Comment:
Currently, a team created with the same name or same numbers will return 409 and not 400 as it could be expected. If this is ok for us I'd leave this as-is since it would require parsing the error and probably handling different cases for pg and sqlite I guess. This will affect webapp depending on the error we want to show, but we could eventually iterate in a second moment. \cc @warenlg

Reply:
Absolutely, thanks for letting me know.

Reply:
+1